scripts/include_list.php

```diff
@@ -45,7 +45,6 @@
 <form action="" method="GET" id="del">
   <h2>Custom Species List</h2>
   <select name="species[]" id="value2" multiple size="30">
-    <option selected value="base">Please Select</option>
```

Comment:
This dummy option is workaround for Firefox where `size` on an empty `<select>` does not correctly size.
Can we add it back in?

Reply:
Thanks I had removed it as otherwise on mobile it always selected it and it caused some issue. Alternative is "disabled" would that be ok with you ?

Reply:
That would be fine.

note that this comment applies only to line 48, on line 24, you _can_ remove the `<option>`, it makes no sense to have it there, as that `<select>` does get filled. I guess that is an error in the original code

Reply:
Thanks ; I've removed the line 24 ; and changed the option from <select> to disabled , hope it will still enable the Firefox workaround.

The advantage of disabled is that on mobile, it is less convenient to first unselect the "Please select" option before selecting another one. I remember that initially it caused some issues on my ios system